### PR TITLE
Favicon & Page Names!

### PR DESCRIPTION
**Pull request by joshrsamuel**
# Problem
Our website currently only displays a favicon on some of our pages, and we only display a page name on the home page. This makes the website feel rough around the edges, and it also makes it harder for users with screen readers to navigate our site.

## Fix
Each page now has the EmpathyBytes logo as its favicon, and it's page name is the same as the title of the page. This should make the website feel more consistent as well as make it more accessible.

## Pictures
The pictures below display how the Experiences page on the website looked like in the browser tab UI, before and after this change.

### Before
image

### After
image

## Minor Accessibility Improvement:
* Added `alt` attribute to the image in `src/pages/webteam.js` to improve accessibility.
* More images will receive alt attributes as I work on code cleanup, this is just one of the ones I caught while working on this ticket.

### CODE REVIEW

File: src/pages/webteam.js
Comment (AmaraWilliams) [inline src/pages/webteam.js:1]: I think you should structure the header and alt file for webteam.js the same why you did for the other files. I noticed your changes had no effect for this specific page.
image

I imagine adding this code might fix the problem:
image

**Review by AmaraWilliams — CHANGES_REQUESTED** [review]
Nice work! It's a quick fix I think for the page title

File: src/pages/webteam.js
Comment (joshrsamuel) [reply src/pages/webteam.js:1]: Good spot! Just fixed it!

**Review by husnajakeer — COMMENTED** [review]
All of these changes look great! I am now able to see the favicon display across all pages.

**Review by husnajakeer — APPROVED** [review]
All of the correct page names now display correctly! Thank you for fixing this Josh!

**Review by AmaraWilliams — APPROVED** [review]
Looks good! All pages have a title and favicon now!

**Pull request merged**